### PR TITLE
Improv/fr locale

### PR DESCRIPTION
**Pull request by ptsakyrellis**
- Added additional test data
- Corrected some typos / Errors

### CODE REVIEW

File: Resources/Locales/fr.json [hunk 1]
```diff
@@ -9,18 +9,26 @@
...
         ],
         "street_prefix": [
-          "Allée, Voie",
-          "Rue",
-          "Avenue",
-          "Boulevard",
+          "Allée",
+          "Avenue",		  
+          "Boulevard",		  
+		  "Chemin",
```
Comment (vadymmarkov) [inline Resources/Locales/fr.json]: @ptsakyrellis Not very critical, but could you please align it? :smile:

**Comment by ptsakyrellis** [comment]
Of course, i can, let me update this

**Comment by ptsakyrellis** [comment]
Should be correctly aligned now !

**Comment by vadymmarkov** [comment]
@ptsakyrellis Awesome! Thanks for your contribution!

**Pull request merged**